src/proof/lean/lean_post_processor.cpp

```diff
@@ -1304,6 +1305,75 @@ bool LeanProofPostprocessCallback::update(Node res,
                   *cdp);
       break;
     }
+    case PfRule::ARITH_MULT_SIGN:
+    {
+      std::vector<Node> variables;
+      std::vector<Node> pols;
+      size_t numArgs = args.size();
+      // last element is the conclusion
+      for (size_t i = 0; i < numArgs - 1; i++)
+      {
+        Assert(arg[i].getKind() == Kind::NOT ||
+               arg[i].getKind() == Kind::LT  ||
+               arg[i].getKind() == Kind::GT);
```

Comment:
```suggestion
        Assert(arg[i].getKind() == Kind::NOT || arg[i].getKind() == Kind::LT
               || arg[i].getKind() == Kind::GT);
```
this clangs formats like this for me